Utils/PersistentDataStore.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using UnityEngine;
+
+namespace DUCK.Utils
```

Comment:
Do you think utils is the right place for this or does it deserve it's own feature namespace. It's pretty small so I'm on the fence, I'd like to yours and other's thoughts

Reply:
Just to clarify that's intended ?

Reply:
Yeah - I thought since it's just a single class, it felt about the same 'weight' as any other Util.